<launch><arg default="0.5" name="col_tolerance" /><param name="number_of_robots" type="int" value="3" /><param name="collision_tolerance" type="double" value="$(arg col_tolerance)" /><arg default="one_robot_rwp.launch" name="task_controller" /><group ns="R1"><param name="tf_prefix" value="robot1_tf" /><include file="$(find random_control)/launch/$(arg task_controller)"><arg name="id" value="1" /><arg name="robot_name" value="R1" /><arg name="robot_model" value="waffle" /><arg name="rnd_seed" value="7770" /><arg name="yaw0" value="0.0" /><arg name="x0" value="1.0" /><arg name="y0" value="1.0" /></include></group><group ns="R2"><param name="tf_prefix" value="robot2_tf" /><include file="$(find random_control)/launch/$(arg task_controller)"><arg name="id" value="2" /><arg name="robot_name" value="R2" /><arg name="robot_model" value="waffle" /><arg name="rnd_seed" value="4412" /><arg name="yaw0" value="0.0" /><arg name="x0" value="2.0" /><arg name="y0" value="2.0" /></include></group><group ns="R3"><param name="tf_prefix" value="robot3_tf" /><include file="$(find random_control)/launch/$(arg task_controller)"><arg name="id" value="3" /><arg name="robot_name" value="R3" /><arg name="robot_model" value="waffle" /><arg name="rnd_seed" value="2597" /><arg name="yaw0" value="0.0" /><arg name="x0" value="3.0" /><arg name="y0" value="3.0" /></include></group></launch>